cgogn/core/cmap/map_base.h

```diff
@@ -168,6 +167,23 @@ class MapBase : public MapBaseData<MAP_TRAITS>
 		return idx;
 	}
 
+	template<Orbit ORBIT>
+	inline void compact_orbit_container()
+	{
+		if (!this->template is_embedded<ORBIT>())
+			return;
+
+		auto& cac = this->template get_attribute_container<ORBIT>();
+		const std::vector<unsigned int>& map_old_new = cac.template compact<ConcreteMap::PRIM_SIZE>();
+		this->parallel_foreach_dart([&map_old_new,this](Dart d, uint32)
+		{
+			uint32& old_idx = this->embeddings_[ORBIT]->operator[](d);
+			const uint32 new_idx = map_old_new[old_idx];
+			if (new_idx != UINT32_MAX)
+				old_idx = new_idx;
+		});
+	}
+
```

Comment:
Le retour du compactage. Je pensai qu'on avait renoncer à cause du danger ?
Si on garde ça, il faut absolument prévenir l'utilisateur que cette méthode invalide tous les itérateurs sur l'orbite, les caches et autres structures annexes propre à l'application. C'est particulièrement sensible dans le cas multi-thread.
N'est pas redondant avec la gestion des indices dédiés au calcul matriciel dont on a parlé dans Gitter ?